test/HTS.t.sol

```diff
@@ -54,4 +54,67 @@ contract HTSTest is Test, TestSetup {
+        assertEq(tokenInfo.token.symbol, "USDC");
+        assertEq(tokenInfo.token.treasury, address(0x0000000000000000000000000000000000001438));
+        assertEq(tokenInfo.token.memo, "USDC HBAR");
+        assertEq(tokenInfo.token.tokenSupplyType, false);
+        assertEq(tokenInfo.token.maxSupply, 0);
+        assertEq(tokenInfo.token.freezeDefault, false);
+        assertEq(tokenInfo.token.tokenKeys.length, 7);
+        // AdminKey
+        assertEq(tokenInfo.token.tokenKeys[0].keyType, 0x1);
+        assertEq(tokenInfo.token.tokenKeys[0].key.inheritAccountKey, false);
+        assertEq(tokenInfo.token.tokenKeys[0].key.contractId, address(0));
+        assertEq(tokenInfo.token.tokenKeys[0].key.ed25519, vm.parseBytes("5db29fb3f19f8618cc4689cf13e78a935621845d67547719faf49f65d5c367cc"));
+        assertEq(tokenInfo.token.tokenKeys[0].key.ECDSA_secp256k1, bytes(""));
+        assertEq(tokenInfo.token.tokenKeys[0].key.delegatableContractId, address(0));
+        // FreezeKey
+        assertEq(tokenInfo.token.tokenKeys[2].keyType, 0x4);
+        assertEq(tokenInfo.token.tokenKeys[2].key.inheritAccountKey, false);
+        assertEq(tokenInfo.token.tokenKeys[2].key.contractId, address(0));
+        assertEq(tokenInfo.token.tokenKeys[2].key.ed25519, vm.parseBytes("baa2dd1684d8445d41b22f2b2c913484a7d885cf25ce525f8bf3fe8d5c8cb85d"));
+        assertEq(tokenInfo.token.tokenKeys[2].key.ECDSA_secp256k1, bytes(""));
+        assertEq(tokenInfo.token.tokenKeys[2].key.delegatableContractId, address(0));
+        // SupplyKey
+        assertEq(tokenInfo.token.tokenKeys[4].keyType, 0x10);
+        assertEq(tokenInfo.token.tokenKeys[4].key.inheritAccountKey, false);
+        assertEq(tokenInfo.token.tokenKeys[4].key.contractId, address(0));
+        assertEq(tokenInfo.token.tokenKeys[4].key.ed25519, vm.parseBytes("4e4658983980d1b25a634eeeb26cb2b0f0e2e9c83263ba5b056798d35f2139a8"));
+        assertEq(tokenInfo.token.tokenKeys[4].key.ECDSA_secp256k1, bytes(""));
+        assertEq(tokenInfo.token.tokenKeys[4].key.delegatableContractId, address(0));
+        // Expiry
+        assertEq(tokenInfo.token.expiry.second, 1706825707000718000);
+        assertEq(tokenInfo.token.expiry.autoRenewAccount, address(0));
+        assertEq(tokenInfo.token.expiry.autoRenewPeriod, 0);
+        assertEq(tokenInfo.totalSupply, 10000000005000000);
+        assertEq(tokenInfo.deleted, false);
+        assertEq(tokenInfo.defaultKycStatus, false);
+        assertEq(tokenInfo.pauseStatus, false);
+        assertEq(tokenInfo.fixedFees.length, 0);
+        assertEq(tokenInfo.fractionalFees.length, 0);
+        assertEq(tokenInfo.royaltyFees.length, 0);
+        assertEq(tokenInfo.ledgerId, testMode == TestMode.FFI ? "0x01" : "0x00");
+    }
+
+    function test_HTS_getTokenInfo_should_revert_for_invalid_token() external {
+        address token = address(0);
+        vm.expectRevert(bytes("getTokenInfo: invalid token"));
+        HtsSystemContract(HTS_ADDRESS).getTokenInfo(token);
```

Comment:
Is this the behavior of HTS? Maybe here the `responseCode` is used.

We could also include another case with `makeAddr` in addition to `address(0)`.